libs/translator/reader/utils.h

```diff
@@ -425,7 +425,7 @@ static inline bool VtValueGetBool(const VtValue& value)
         VtArray<long> array = value.UncheckedGet<VtArray<long>>();
         return array.empty() ? false : (array[0] != 0);   
     }
-    return value.Get<bool>();
+    return false;
```

Comment:
something that would be great is to pass the default value as an argument:
```
VtValueGetBool(const VtValue &value, bool default=false) 
{ 
   ...
    return default;
}
```
This has the benefit of exposing the fact that the function returns a default if the value is not found ... and later on be able to change the default, as sometimes we might want true and other times false. Same for the other similar functions. 
What do you think ?